pkg/execution/jsonvaluetype.go

```diff
@@ -5,9 +5,11 @@ import (
 	"bytes"
 	"fmt"
 	"io"
+	"strconv"
+	_ "strings"
```

Comment:
what if we remove this ? same for below escape import.

Reply:
done